certified-connectors/FactSet/Readme.md

```diff
@@ -0,0 +1,31 @@
+# FactSet Connector
+
+Prepare for client meetings using the FactSet connector to access key company information in a custom branded, presentation ready PDF. Stay on top of relevant news and events for companies of interest and highlight important topics to drive your discussion with FactSet Signals. Alternatively, use Digital Cards to collaborate with colleagues and send summaries of actionable insights via Teams.
+

```

Comment:
@krithsai Please refer the templates provided for readme.md file and add "publisher" section.

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md